addon/templates/components/frost-password.hbs

```diff
@@ -1,28 +1,62 @@
-<div class='row'>
-  {{frost-text
-    id=componentId
-    value=value
-    class=class
-    type=type
-    onBlur=(action 'onBlur')
-    onInput=(action "onInput")
-    tabindex=tabindex
-    disabled=disabled}}
-  {{#if revealable}}
-    {{#if isRevealerVisible}}
-      {{#frost-button
-        size='none'
-        priority='none'
-        tabIndex='-1'
-        onClick=(action "toggleReveal")}}
-        {{frost-icon icon=revealIcon}}
-      {{/frost-button}}
-    {{/if}}
-  {{/if}}
+{{frost-text
+  autofocus=autofocus
+  class='frost-password-input'
+  disabled=disabled
+  form=form
+  maxlength=maxlength
+  placeholder=placeholder
+  readonly=readonly
+  required=required
+  selectionDirection=selectionDirection
+  tabindex=tabindex
+  title=title
+  type=type
+  value=value
 
-  {{#if isCapsAndReveal}}
-    <div class='reveal-warning'>Caps on</div>
-  {{else if isCapsOn}}
-    <div class='warning'>Caps on</div>
-  {{/if}}
-</div>
+  eventVersion=eventVersion
+  onChange=_eventProxy.change
+  onClick=_eventProxy.click
+  onContextMenu=_eventProxy.contextMenu
+  onDoubleClick=_eventProxy.doubleClick
+  onDrag=_eventProxy.drag
+  onDragEnd=_eventProxy.dragEnd
+  onDragEnter=_eventProxy.dragEnter
+  onDragLeave=_eventProxy.dragLeave
+  onDragOver=_eventProxy.dragOver
+  onDragStart=_eventProxy.dragStart
+  onDrop=_eventProxy.drop
+  onFocusIn=_eventProxy.focusIn
+  onFocusOut=_eventProxy.focusOut
+  onInput=_eventProxy.input
+  onKeyDown=_eventProxy.keyDown
+  onKeyPress=_eventProxy.keyPress
+  onKeyUp=_eventProxy.keyUp
+  onMouseDown=_eventProxy.mouseDown
+  onMouseEnter=_eventProxy.mouseEnter
+  onMouseLeave=_eventProxy.mouseLeave
+  onMouseMove=_eventProxy.mouseMove
+  onMouseUp=_eventProxy.mouseUp
+  onSubmit=_eventProxy.submit
+  onTouchCancel=_eventProxy.touchCancel
+  onTouchEnd=_eventProxy.touchEnd
+  onTouchMove=_eventProxy.touchMove
+  onTouchStart=_eventProxy.touchStart
+
+  onEnter=_eventProxy.enter
+  onEscape=_eventProxy.escape-press
```

Comment:
I think that works, I'd be tempted to more more rebellious and not make it backwards compatible but I guess that'd make me an 🐴 (closest emoji to a donkey I can find).